### PR TITLE
Localization improvements

### PR DESCRIPTION
**Pull request by drobee**
Added translation functions instead of simple strings.

### CODE REVIEW

**Comment by stepanenko3** [comment]
@drobee,
Huge thanks

File: resources/js/pages/Tool.vue [hunk 1]
```diff
@@ -88,22 +88,22 @@
             />
 
             <div class="w-full md:w-3/5 md:px-3 mb-2 md:mb-0">
-                <input type="text" v-model="customCommand.command" placeholder="Enter a Command..." class="w-full form-control form-input form-input-bordered">
+                <input type="text" v-model="customCommand.command" :placeholder="this.__('Enter a Command...')" class="w-full form-control form-input form-input-bordered">
```
Comment (arttemiuss43) [inline resources/js/pages/Tool.vue:91]: Is there a particular reason for using `this` in the template? I think it should be removed.
```suggestion
                <input type="text" v-model="customCommand.command" :placeholder="__('Enter a Command...')" class="w-full form-control form-input form-input-bordered">
```

**Review by arttemiuss43 — CHANGES_REQUESTED** [review]

**Comment by stepanenko3** [comment]
@drobee, We have a conflict

**Review by stepanenko3 — APPROVED** [review]

**Comment by drobee** [comment]
@stepanenko3 you were too fast :) It still needs to be built, because the compiled js contains only the long-running command changes.

**Comment by stepanenko3** [comment]
@drobee, Thanks for warning me

Fixed in [4.2.2](https://github.com/stepanenko3/nova-command-runner/releases/tag/v4.2.2)

**Comment by drobee** [comment]
@stepanenko3 thank you :) Would you add "HACKTOBERFEST-ACCEPTED" label to this PR please? This year I might have time to actually participate. Thanks

**Pull request merged**